prowler/providers/aws/services/elasticache/elasticache_redis_cluster_below_v6_auth_enabled/elasticache_redis_cluster_below_v6_auth_enabled.metadata.json

```
@@ -0,0 +1,32 @@
+{
+  "Provider": "aws",
+  "CheckID": "elasticache_redis_cluster_below_v6_auth_enabled",
+  "CheckTitle": "Ensure Elasticache Elasticache Redis replication groups under version 6.0 have Redis OSS AUTH Enabled.",
```

Please, don't hardcore versions in the metadata, it could change in the future.
```suggestion
  "CheckTitle": "Ensure Elasticache Elasticache Redis replication groups of earlier versions should have Redis OSS AUTH enabled.",
```